characters.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// Character of Kaamelott to retrieve sentences from
+type Character struct {
+	Sentences []string `json:"sentences"`
+}
+
+func loadCharacters() {
+	for index := range Configuration.Characters.List {
+		jsonFile, err := os.Open(Configuration.Characters.Folder + "/" + Configuration.Characters.List[index].File)
+
+		if err != nil {
+			fmt.Println("Error getting " + Configuration.Characters.List[index].Name + "'s sentences")
+		} else {
+			byteValue, _ := ioutil.ReadAll(jsonFile)
+			json.Unmarshal(byteValue, &Configuration.Characters.List[index].Data)
+			fmt.Println(Configuration.Characters.List[index].Name + " loaded succesfully!")
+		}
+
+		jsonFile.Close()
+	}
+}
+
+func displayAvailableCharacters(s *discordgo.Session, m *discordgo.MessageCreate) {
+	message := ""
+	message += "\nLe caca des pigeons, c'est caca. Si tu parle d'un de mes amis, je te dirai ce qu'il a dit."
+	message += "\nJ'ai pleins d'amis à Kaamelott:"
+	for _, character := range Configuration.Characters.List {
+		message += "\n- " + character.Name
+	}
+	s.ChannelMessageSend(m.ChannelID, message)
+}
+
+func handleCalledCharacter(s *discordgo.Session, m *discordgo.MessageCreate) {
+	for _, character := range Configuration.Characters.List {
+		if strings.Index(strings.ToUpper(m.Content), strings.ToUpper(character.Name)) > -1 {
+			if len(character.Data.Sentences) < 1 {
+				s.ChannelMessageSend(m.ChannelID, "Mordu, mordu mordu moooooooooooooooordu mordu mordu mordu mordu mordu mordu mordu mordu mordu mordu mordu morduuuuuuuuuuuuuuuuuuuuuuuuuuuuu!!!!")
+				return
+			}
+
+			index := rand.Intn(len(character.Data.Sentences))
+
+			s.ChannelMessageSend(m.ChannelID, character.Data.Sentences[index])

```

Comment:
how is a new line handled ? is there a way to add multi line quotes ?

Reply:
In the quote itself, "\n" is used